examples/cms-hashnode/components/integrations.tsx

```diff
@@ -0,0 +1,115 @@
+import { useEffect } from 'react'
+import { useAppContext } from './contexts/appContext'
+
+export function Integrations() {
+  const { publication } = useAppContext()
+  const {
+    gaTrackingID,
+    fbPixelID,
+    hotjarSiteID,
+    matomoURL,
+    matomoSiteID,
+    fathomSiteID,
+    fathomCustomDomain,
+    fathomCustomDomainEnabled,
+    plausibleAnalyticsEnabled,
+  } = publication.integrations ?? {}
+  const domainURL = new URL(publication.url).hostname
+
+  let fbPixel = `
```

Comment:
Could we strip this example back down to the most minimal example following `cms-starter` + Hashnode?

Reply:
@leerob I have now removed the third party integrations and analytics tracking

Reply:
@leerob Could someone from the team please check this?

Reply:
@leerob Could you please provide more insight into why some jobs are failing in this build? The current build results lack sufficient detail for us to diagnose the issue. Locally, both `pnpm build` and `pnpm lint`
<img width="1063" alt="Screenshot 2024-06-26 at 12 48 36 PM" src="https://github.com/vercel/next.js/assets/13347449/c648163e-4ccb-40ef-962c-7a1777062843">
<img width="1071" alt="Screenshot 2024-06-26 at 12 49 09 PM" src="https://github.com/vercel/next.js/assets/13347449/e5e9fbc0-dcef-49cd-94b4-db14dc6e4d16">
 commands run successfully without any errors (Screenshot attached for reference).